src/Stratis.Bitcoin.Tests.Common/TestBase.cs

```diff
@@ -25,31 +25,9 @@ public TestBase(Network network)
 
         public static string AssureEmptyDir(string dir)
         {
-            int deleteAttempts = 0;
```

Comment:
Would it be possible / desirable to clean folders from previous runs ? I can help you write the code if the is something we want to do 👍